src/app/ui-kit/button/button.component.html

```diff
@@ -0,0 +1,15 @@
+<button class="m-1 px-5 rounded border-2 flex items-center" [ngClass]="[
+  size === 'sm' ? prev || next ? 'py-[2.5px]' : 'py-1.5' : 'py-3',
+  prev ? 'pl-2' : next ? 'pr-2' : '',
+  type === 'accent' && (state === 'enabled' || state === 'hover') ? 'bg-accent' : state === 'disabled' ? 'bg-secondary' : 'bg-transparent',
+  type === 'ghost' && (state === 'enabled' || state === 'hover') ? 'border-accent' : state == 'disabled' ? 'border-secondary': 'border-transparent',
+  (type == 'ghost' && state == 'disabled') ?  'bg-transparent' :'',
+  state === 'enabled' ? 'text-bright' : state === 'disabled' ? 'text-shade' : state === 'hover' ? 'text-error' : ''
+]">
+  <img *ngIf='prev' [src]="'../../..//assets/images/chevron-icons/chevron-left-' + state + '.png'" alt="chevron icon" />
+  <app-typography fontWeight='font-[600]' fontSize="text-[14px]" lineHeight="leading-[18px]">
```

Comment:
check tailwind docs to prevent adding arbitrary value in case tailwind has what you need.
For example: font-[600] can be applied as font-semibold https://tailwindcss.com/docs/font-weight